AirBench/Scripts/Home.js

```diff
@@ -60,6 +72,36 @@
     });
     map.on('click', clickMarker);
     //map.addEventListener('click', clickMarker);
+    function updateList(){
+        let min = document.getElementById('min').value;
+        let max = document.getElementById('max').value;
```

Comment:
I'd suggest that you use `parseInt` to convert these values to numbers before doing the comparisons below.